src/models/subscriptiondata.cpp

```diff
@@ -50,12 +50,13 @@ bool SubscriptionData::fromSettings() {
 
   logger.debug() << "Reading the subscription data from settings";
 
-  const QByteArray& json = settingsHolder->devices();
+  const QByteArray& json = settingsHolder->subscriptionData();
   if (json.isEmpty() || !fromJsonInternal(json)) {
     return false;
   }
 
   m_rawJson = json;
+  emit changed();
```

Comment:
One concern is that it hasn't really changed, and does not follow the pattern in user.cpp, serverdata.cpp etc, where the signal is emitted only on an actual change.

Reply:
I suppose it depends on how you look at it

On one hand, all of the `Q_PROPERTY`'s that rely on this `changed()` signal being emitted ARE being changed. For example, if we were to load a UI that used this data before this subscriptiondata object was initialized, that UI would not update to reflect the correct values after subscriptiondata initialization

On the other hand, the subscription data on launch is being pulled from settings, so it would be the same data as the last session. In that respect, the data hasn't change

I don't see any particular drawbacks to emitting the signal here (are there any other places we catch this signal?), but I do also dislike the inconsistency with the other data model classed. What do you think about just creating a different signal. Maybe `Initialized()`?

Reply:
initialized sounds good.